### DISCUSSION TITLE
Agent submition without registration

### DISCUSSION THREAD

**Discussion by kronga** [yasserfarouk/scml#59] [Q&A] [answered]
Hi, 

We developed an agent for the OneShot track. However, we can't find a way to submit it.
I understand that the registration deadline is over, but it is stated that registration is only optional.
How can we submit our agent without registering? 

Thanks,
David

**Comment by yasserfarouk** [comment, marked as answer]
Hi
The registration deadline was extended to July 10th.

The server will be reopened for registration in a few minutes. 

Thanks
